### PR TITLE
[ENH] Datasets tools to crate and check info file

### PR DESCRIPTION
**Pull request by BlazZupan**
datasets-info-create.py takes a remote data file and walks the user through the items to add to the info file. make sure you check the info file later.

datasets-info-check.py checks the info file. It is needed only if the info file was created manually.

### CODE REVIEW

File: tools/datasets-info-create.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+import readline
+import shlex
+import subprocess
+import textwrap
+import itertools
+from serverfiles import LocalFiles, ServerFiles
+from Orange.misc.environ import data_dir
+
+# url = "/Users/blaz/Desktop/grades.tab"
+# url = "http://file.biolab.si/datasets/grades.tab"
```
Comment (ajdapretnar) [inline tools/datasets-info-create.py:21]: Should these two lines be here?

File: tools/datasets-info-create.py [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+info["title"] = query("title", "Title is composed of one or few words. Use sentence case.")
+info["year"] = query("year", "Year of first publication of the data set.")
+info["collection"] = query("collection", "Collection name, like UCI, R, or GEO.")
+
+source = query("Source", "Source description (like UCI ML Repository, or KDD Cup Competitions")
+if not source:
+    info["source"] = None
+else:
+    source_url = query("Source URL", "URL of the source (required), like http://orange.biolab.si")
+    info["source"] = "<a href={}>{}</a>".format(source, source_url)
```
Comment (kernc) [inline tools/datasets-info-create.py:147]: Quote attribute value.

**Comment by kernc** [comment]
Seems like this script better belongs into https://github.com/biolab/datasets?

File: tools/datasets-info-create.py [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+    extra=pp_tags
+)
+if tag_s:
+    info["tags"] = [candidates[int(t)] if t.isdigit() else t for t in tag_s.split(" ")]
+else:
+    info["tags"] = []
+
+# url
+if file_is_local or args.upload:
+    cmd = "scp {} file:~/anonymous/datasets/".format(args.file_name)
```
Comment (kernc) [inline tools/datasets-info-create.py:187]: How does this work?

**Pull request merged**